src/components/job-common.js

```diff
@@ -172,22 +173,52 @@ export const getTaskCost = ({ vmCostUsd, taskStartTime, taskEndTime }) => {
 };
 
 export const renderTaskCostElement = (cost) => {
-  if (cost === 0.0) {
+  if (cost.toFixed(2) === '0.00') {
     return '< $0.01';
   }
   return `$${cost.toFixed(2)}`;
 };
 
-export const calculateTotalCost = (callObjects) => {
+export const calculateTotalCost = async (callObjects, loadForSubworkflows) => {
   let total = 0;
-  Object.values(callObjects).forEach((call) => {
-    if (!call[0].taskStartTime) {
-      total += 0;
-    } else if (call[0].taskEndTime) {
-      total += getTaskCost({ vmCostUsd: call[0].vmCostUsd, taskStartTime: call[0].taskStartTime, taskEndTime: call[0].taskEndTime });
-    } else {
-      total += getTaskCost({ vmCostUsd: call[0].vmCostUsd, taskStartTime: call[0].taskStartTime });
+  let callObs = callObjects;
+
+  if (!_.isNil || callObjects?.subWorkflowId) {
+    const subWorkflows = await loadForSubworkflows(callObjects.subWorkflowId);
+    callObs = subWorkflows?.calls;
```

Comment:
my personal opinion... this conditional reassignment of `callObs` (along with the similarity in naming to `callObjects`) is confusing enough that it deserves a comment